sirepo_bluesky/sirepo_bluesky.py

```diff
@@ -89,12 +91,14 @@ def copy_sim(self, sim_name):
 
     def delete_copy(self):
         """ Delete a simulation which was created using copy_sim(). """
-        assert self.is_copy
+        if not self.is_copy:
+            raise NotImplementedError
         res = self._post_json('delete-simulation', {
             'simulationId': self.sim_id,
             'simulationType': self.sim_type,
         })
-        assert res['state'] == 'ok'
+        if not res['state'] == 'ok':
+            raise NotImplementedError
```

Comment:
Here it would be useful to see the value of `res['state']` and maybe some other parameters from `res` (HTTP status code maybe).